9.1.индексация/9.1.11.py

```diff
@@ -0,0 +1,7 @@
+word = input()                  
+message = "Цифр нет"                
+for i in range(len(str(word))):  
+    if word[i] in "0123456789":   
+        message = "Цифра"           
+        break                  
```

Comment:
Братан, усложнил очень :) Зачем пробегать по индексам, когда строки являются последовательностями, значит по ним самим можно проходить:
```suggestion
for char in word:  
    if char in "0123456789":   
        message = "Цифра"           
        break                  
```